### PR TITLE
Add Devil's Reef cards

### PR DESCRIPTION
**Pull request by zzorba**

### CODE REVIEW

File: pack/tic/def.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"illustrator": "Imad Awan",
+		"name": "Radiant Smite",
+		"pack_code": "def",
+		"position": 153,
+		"quantity": 2,
+		"skill_combat": 1,
+		"skill_willpower": 1,
+		"text": "<b>Fight.</b> You man use [willpower] instead of [combat] for this attack. When you initiate this attack, search the chaos bag for up to 3 [bless] tokens, and seal them here. For each [bless] token sealed on Radiant Smite, you get +1 skill value and deal +1 damage for this attack. If this attack defeats the attacked enemy, return the sealed tokens to the token pool. Otherwise, release them.",
```
Comment (drawntotheflame) [inline pack/tic/def.json:14]: 'may' not 'man' early on.

File: pack/tic/def.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		"cost": 1,
+		"deck_limit": 2,
+		"faction_code": "neutral",
+		"illustrator": "Imad Awan",
+		"name": "Manipulate Destiny",
+		"pack_code": "def",
+		"position": 162,
+		"quantity": 2,
+		"skill_wild": 1,
+		"text": "Reveal tokens from the chaos bag until you reveal a [curse], [auto_fail], [bless], or [elder_sign] token. If you revealed...\n- ... a [curse] or [auto_fail] token, deal 2 damage to an enemy at your location.\n- ... a [bless] or [curse] token, heal 2 damage from an investigator or [[Ally]] asset at your location.\nThis action does not provoke attacks of opportunity.",
```
Comment (drawntotheflame) [inline pack/tic/def.json:65]: Please can you close up the space between ... and 'a' after the \n: '\n ...a [curse]' and '\n ...a [bless]'

Comment (drawntotheflame) [reply pack/tic/def.json:65]: Also, for the second bullet, you've put [curse] instead of [elder_sign].

File: pack/tic/def.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		"traits": "Spell.",
+		"type_code": "event",
+		"xp": 2
+	}
+]
```
Comment (drawntotheflame) [inline pack/tic/def.json:70]: Do you want to add 'The Truth Beckons' from the card fan? Number 07154, seeker, 1 cost, 0xp, Insight event, int + agi icons, illustrator Greg Bobrowski. Text (you'll need to put the proper bold tag in):
'Play only if you are not engaged with an enemy.\n[b]Move.[/b] Choose an unrevealed location. Move (one location at a time) along the shortest path toward that location, until you enter that location. End this effect if you reveal a location, if an enemy engages you, or if your movement is blocked.'

**Comment by zzorba** [comment]
All done, can be merged @Kamalisk

**Pull request merged**